dev-packages/e2e-tests/test-applications/nextjs-14/instrumentation.ts

```diff
@@ -0,0 +1,13 @@
+import * as Sentry from '@sentry/nextjs';
+
+export function register() {
+  if (process.env.NEXT_RUNTIME === 'nodejs' || process.env.NEXT_RUNTIME === 'edge') {
```

Comment:
This begs a question: Suppose I have the same config for node and edge (as in this example). Do I even need the `if` statement? Asking because I guess this is a common thing for users and if it's indeed necessary, we should point that out.